NavKit/CustomizableNavigation.swift

```diff
@@ -73,6 +78,12 @@ public extension CustomizableNavigation where Self: UIViewController, Self: UIGe
 
             navigationBar?.subviews.first?.insertSubview(barView!, at: 0)
         }
+        
+        if #available(iOS 11.0, *) {
+            if largeNavBarTitle {
+                navigationBar.prefersLargeTitles = true
```

Comment:
Nice addition Sai! 😊 One thing to note, when people change the property (let's say `backImage`) when the screen has been appeared (e.g when responding to some UI events), people normally call `self.updateNavigation()` and expect an instant change.

And by this code, I think it isn't possible to see a change of `largeNavBarTitle` because you always set it to `true`. Shouldn't it be `navigationBar.prefersLargeTitles = largeNavBarTitle`? (without the if).

Let me know what you think! 🙏🏻

Reply:
Any thoughts on this @theindiandev1065? 😊